less/navigation.less

```diff
@@ -1 +1,51 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.navContainer {
+    width: 100%;
+    background: @wild-sand;
+    color: @nav-font;
+    border-bottom: 1px solid @nav-border;
+    padding: 1% 0;
+
+        .smallerNavContainer {
```

Comment:
try using more descriptive class names. What is this actually containing. just the nav? the nav elements? is it wrapper a smaller nav or is it the smaller nav itself?